graphdatascience/session/gds_sessions.py

```diff
@@ -86,6 +89,7 @@ def get_or_create(
         db_connection: DbmsConnectionInfo,
         ttl: Optional[timedelta] = None,
         cloud_location: Optional[CloudLocation] = None,
+        timeout: Optional[int] = 300,
```

Comment:
lets default to None and pass the optional also to the impl. 
Propose to have the optional to math.Inf only on the aura_api level